Sources/VRage.Dedicated/Configurator/ConfigForm.cs

```diff
@@ -69,7 +69,7 @@ private void startButton_Click(object sender, EventArgs e)
             else // Local / Console
             {
                 string[] cmdLine = Environment.GetCommandLineArgs();
-                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? cmdLine[1] : "" ) + " -console -ignorelastsession");
+                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? String.Join(" ", cmdLine, 1, cmdLine.Length - 1 ) : "") + " -console -ignorelastsession");
```

Comment:
According to [`ProcessStartInfo.Arguments`](https://msdn.microsoft.com/en-us/library/system.diagnostics.processstartinfo.arguments%28v=vs.110%29.aspx) arguments that contain strings need to be surrounded by quotes. And arguments containing quotes need to be escaped. 

Also, what happens if one of the `cmdLine` arguments is `-console` or `-ignorelastsession`? Can the started process handle duplicate command line arguments?